libpod/image/image.go

```diff
@@ -615,14 +615,14 @@ func (i *Image) TopLayer() string {
 // TODO: the force param does nothing as of now. Need to move container
 // handling logic here eventually.
 func (i *Image) Remove(ctx context.Context, force bool) error {
-	parent, err := i.GetParent(ctx)
-	if err != nil {
-		return err
```

Comment:
If you just log.Warnf(err) here, everything else will work fine, I believe.

Reply:
Tried it, and it still failed with another file-not-found I haven't chased down yet.  It's mixed in with another error, so trace logging still only gives me a backtrace that points to `pkg/errorhandling.JoinErrors()`.  Frustrating, that.

Reply:
Never mind, `rmi -a` apparently doesn't use the same code path as `rmi`.